includes/fbutils.php

```diff
@@ -445,6 +445,17 @@ public static function get_store_name() {
 			return ( self::$store_name ) ? ( self::$store_name ) : 'A Store Has No Name';
 		}
 
+		public static function get_store_url() {
```

Comment:
not sure why this is needed

Reply:
I need the store url for the campaign to be created. 
So I first try to get the site_url, if it doesn't work I try with hostname